rmw_zenoh_cpp/src/detail/rmw_context_impl_s.cpp

```diff
@@ -419,13 +436,10 @@ class rmw_context_impl_s::Data final
 };
 
 ///=============================================================================
-static void graph_sub_data_handler(const z_sample_t * sample, void * data)
+static void graph_sub_data_handler(z_loaned_sample_t * sample, void * data)
```

Comment:
```suggestion
static void graph_sub_data_handler(const z_loaned_sample_t * sample, void * data)
```

Reply:
The same. We've changed the function signature to support the case of "mutably borrow".